README.md

```diff
@@ -94,7 +104,12 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
+someFunction() //this is an example of calling a function, which means you are executing a function you've already defined.
 
+someFunction: function(){
+  some+other+stuff
+}
+someFunction //this is referencing a function, referencing something like a variable that has a function stored in it will not execute the function, but rather return the function itself.
 ```
```

Comment:
:+1:  Calling a function is actually invoking that function to be executed immediately, while referencing is just referring to a particular function, for example, when referencing a function during an event listener.